lib/msal-node/src/utils/Constants.ts

```diff
@@ -47,6 +47,20 @@ export const Constants = {
     JWT_BEARER_ASSERTION_TYPE: 'urn:ietf:params:oauth:client-assertion-type:jwt-bearer'
 };
 
+/**
+ * API Codes for Telemetry purposes. 
+ * Before adding a new code you must claim it in the MSAL Telemetry tracker as these number spaces are shared across all MSALs
+ * 0-99 Silent Flow
+ * 600-699 Device Code Flow
+ * 800-899 Auth Code Flow
+ */
+export enum ApiId {
+    acquireTokenSilent = 62,
+    acquireTokenByCode = 871,
+    acquireTokenByRefreshToken = 872,
+    acquireTokenByDeviceCode = 671
+};
```

Comment:
Are they no API ids for confidential clients? Are we adding it later?

Reply:
Thanks for the callout. Confidential Client wasn't built out yet when this PR was first put up. Will add.